app/models/tickets_edit_person.rb

```diff
@@ -60,4 +60,14 @@ def self.create_ticket(wca_id, changes_requested, requester)
       return ticket
     end
   end
+
+  DEFAULT_SERIALIZE_OPTIONS = {
+    include: %w[tickets_edit_person_fields],
+  }.freeze
+
+  def serializable_hash(options = nil)
+    json = super(DEFAULT_SERIALIZE_OPTIONS.merge(options || {}))
+    json[:class] = self.class.to_s.downcase
```

Comment:
This line is actually not needed. Sorry that I suggested you this change over chat, but I came to know late that this line is not needed.